_posts/2022-09-16-trino-summit-2022-teaser.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Trino Summit 2022 will be cute and legendary"
+author: "Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-tenth-birthday/creators.jpeg
+---
+
+Bun Bun is back and this year we have an exciting lineup of speakers. The range
+of topics span from architectures like data mesh and data lakehouses, to running
+Trino at scale with fault-tolerance and how to use `EXPLAIN`. This conference is
+free and takes place on November 10th. The summit is in-person but also is
+available to virtual attendees as well. Find out more details below!
+
+<!--more-->
+
+## Location
+
+This years Trino Summit will be hosted live at the Commonwealth Club in San 
+Francisco, CA. In-person registration is limited to 250 seats so make sure you
+register quickly before spots run out!
+
+## Announcing the first sessions and speakers
+
+1. Keynote: State of Trino
```

Comment:
Add an entry sentence that mentions that call for speakers closed and we have a full line up .. and here are a first few teasers